AstronomyUITests/AstronomyUITests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  AstronomyUITests.swift
+//  AstronomyUITests
+//
+//  Created by Norlan Tibanear on 9/24/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+
+class AstronomyUITests: XCTestCase {
+
+    private var app: XCUIApplication {
+        return XCUIApplication()
+    }
+    
+    override func setUp() {
+        let app = XCUIApplication()
+        app.launchArguments = ["UITesting"]
+        continueAfterFailure = false
+        app.launch()
+        
+    }
+
+    
+    func testNextSol() {
+        app.launch()
+        
+        app.navigationBars["Sol 1"]/*@START_MENU_TOKEN@*/.buttons["PhotosCollectionViewController.NextSolButton"]/*[[".buttons[\">\"]",".buttons[\"PhotosCollectionViewController.NextSolButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        XCTAssert(app.navigationBars["Sol 2"].exists)
+    }
```

Comment:
Your UI tests could be refactored a bit to make things easier to follow. Making variables that get the buttons would allow your tests to read more like this (mine was sol 16, I know yours is 2):

``` swift
func testNextSolButtonNavigatesToSol16() throws {
    nextSolButton.tap()
    XCTAssertEqual(navBarTitleLabel, "Sol 16")
}
```

Also note that I am putting the expected outcome in the test name. If you can do this, it is a good habit to get into, and I'm sure that you will see Paul talk about it in the Testing Swift book you are reading.